apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt

```diff
@@ -154,9 +182,14 @@ fun ChatListView(chatModel: ChatModel, settingsState: SettingsViewState, setPerf
 }
 
 @Composable
-private fun OnboardingButtons(openNewChatSheet: () -> Unit) {
+private fun OnboardingButtons() {
```

Comment:
OnboardingButtons can be removed as well, I think, as they are no longer shown (we always have at least non deletable local notes) and we can't even test whether new changes haven't broken them